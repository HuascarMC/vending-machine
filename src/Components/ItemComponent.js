import React from 'react';

class ItemComponent extends React.Component {
  render() {
    return(
      <div className="item-wrapper">
        <p className="img-title">bucket</p>
        <div className="order-img">
          <img />
        </div>
      </div>
    );
  }
}

export default ItemComponent;
