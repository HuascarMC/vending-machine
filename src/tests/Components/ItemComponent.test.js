import React from 'react';
import ItemComponent from '../../Components/ItemComponent'
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

it('should render without crashing', () => {
  shallow(<ItemComponent />);
});

it('should render item-wrapper div', () => {
  const wrapper = shallow(<ItemComponent />);
  const div = wrapper.find('.item-wrapper');
  expect(div.exists()).toEqual(true);
});

it('should render p tag title', () => {
  const wrapper = shallow(<ItemComponent />);
  const div = wrapper.find('.title');
  expect(div.exists()).toEqual(true);
});
