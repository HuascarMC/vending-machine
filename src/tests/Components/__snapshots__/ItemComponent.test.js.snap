// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<div
  className="item-wrapper"
>
  <p
    className="title"
  >
    bucket
  </p>
  <div
    className="order-img-container"
  >
    <img
      className="order-img"
      src={undefined}
    />
  </div>
</div>
`;
